### ISSUE TITLE
Alexa method assigned some ICANN TLDs to wrong parties

### ISSUE THREAD

**Issue by dnsguru** [handshake-org/hs-names#10]
tl;dr:  The .WEB and .SPA allocations were missed in the bootstrap development and were caught by the Alexa 100k approach, but were assigned to the wrong parties.

.WEB ended up with Verisign, but Handshake.org's assigned it as claimable by 1&1 IONOS
.SPA ended up assigning it to the Malaysian Government

From #6
### Names that are already reserved:

| name | target (who can claim on HNS) | ICANN App | Issue |
|------|---------------------------------|------|------|
| merck | merck.com | Contested/On Hold | It is Complicated |
|spa| spa.gov.my | [1-1309-81322 (In PDT)](https://gtldresult.icann.org/applicationstatus/applicationdetails/123) | Different: MY Govt?|
|web| web.de | [1-1296-36138 (In Contracting)](https://gtldresult.icann.org/applicationstatus/applicationdetails/1053) | Different: *HNS* 1&1 Ionos, *DNS* Verisign |
|webs| webs.com  | [1-1033-73917 (In Contracting)](https://gtldresult.icann.org/applicationstatus/applicationdetails/1560) | Same/OK : Vistaprint |

### Names that are already owned on HNS by auction winners:
idn, music, xn--jlq480n2rg (.AMAZON in Chinese)

### The only remaining names that could be soft-forked to prevent future auctions are:

| name | rollout block | Application Info |
|-------|-------------|----------------|
| hotel  |  45360 | [HOTEL Top-Level-Domain S.a.r.l](https://gtldresult.icann.org/applicationstatus/applicationdetails/1562) |
| kids    | 26208 | [DotKids Foundation Limited](https://gtldresult.icann.org/applicationstatus/applicationdetails/161) |
|  xn--cckwcxetd | 51408 | [Amazon](https://gtldresult.icann.org/applicationstatus/applicationdetails/929) | 

_Originally posted by @pinheadmz in https://github.com/handshake-org/hs-names/issues/6#issuecomment-635352309_
